owlkettle/widgets.nim

```diff
@@ -1693,6 +1693,10 @@ renderable RadioGroup of BaseWidget:
       for row in state.rows:
         gtk_box_set_spacing(row.box, state.rowSpacing.cint)
   
+  hooks orient:
+    property:
+      gtk_orientable_set_orientation(state.internalWidget, state.orient.toGtk())
```

Comment:
This one is actually a bug-fix: Radio-buttons did not react to if you changed the orientation at runtime because they had no property hook

Reply:
You are right, that seems to be missing :+1: